tests/flytekit/integration/remote/test_remote.py

```diff
@@ -50,6 +50,37 @@ def register():
     assert out.returncode == 0
 
 
+# only accept one arg for now.
+# be careful with different wf_name in each wf.
+def run(file_name, wf_name, arg_key, arg_value):
```

Comment:
you can use `*args` to support multiple args
```suggestion
def run(file_name, wf_name, *args):
```